js/components/utils/hotkeys.vue

```diff
@@ -3,8 +3,10 @@
     @keydown.space="togglePlayback"
     @keydown.j = "playNext"
     @keydown.k = "playPrev"
-    @keydown.f = "search"
-    @keydown.l = "toggleLike"
+    @keydown.h = "playRewind"
```

Comment:
I'd suggest "rewind" and "forward" as method names (this applies to `playback` service as well).